.github/workflows/ci.yml

```diff
@@ -12,33 +12,6 @@ env:
   GOPROXY: https://proxy.golang.org/
 
```

Comment:
Given the split now, shall we call this workflow (both file & `name` above) something like `ci-tests`?

Reply:
Yes! I would just go with `test` then